src/libs/elektra/kdb.c

```diff
@@ -283,25 +233,14 @@ static void ensureContractGlobalKs (KDB * handle, KeySet * contract)
 
 /**
  * Handles the system:/elektra/contract/mountglobal part of kdbOpen() contracts
+ * This function is only necessary for the GOPTS hook.
```

Comment:
Unless I missed something, this function can't actually be needed. All it does now is cut stuff out of `contract` and then delete it.

Reply:
You're right. I've even duplicated the logic within `hooks.c`.